releasing/plugin-commands-publishing/src/pack.ts

```diff
@@ -145,8 +151,15 @@ export async function handler (
   } else {
     packedTarballPath = path.relative(opts.dir, path.join(dir, tarballName))
   }
+  const packedContents = files.sort((a, b) => a.localeCompare(b, 'en'))
+  if (opts.format === 'json') {
+    return JSON.stringify({
+      tarball: packedTarballPath,
+      contents: packedContents,
+    }, null, 2)
```

Comment:
check what `npm pack --json` prints and maybe do something similar. Probably printing `filename` and `files` similar to npm's output should be enough.

Reply:
Yeah, that's how `npm` prints. I have updated it